akka-stream-testkit/src/test/scala/akka/stream/testkit/ChainSetup.scala

```diff
@@ -7,20 +7,23 @@ import org.reactivestreams.Publisher
 import akka.stream.ActorFlowMaterializer
 
 class ChainSetup[In, Out](
-  stream: Flow[In, In] ⇒ Flow[In, Out],
+  stream: Flow[In, In, _] ⇒ Flow[In, Out, _],
```

Comment:
this style is remnant of the old Flow where it was what now Source is. We should refactor these to use via. Probably not in this PR but we should open a ticket.